### DISCUSSION TITLE
Ticket opened for 9 days

### DISCUSSION THREAD

**Discussion by ejgutierrez74** [community/community#37389] [GitHub Education]
Hi, its normal or usual to have an open ticket for more than 9 days ?
Is there any estimated time to be attended.

I cant renew my teacher license, as i cant upload the document because its too small, and also need to manage my students if its possible....

**Comment by samus-aran** [comment]
Hi @ejgutierrez74 

It depends on the application. 🙂  Usually the system can auto verify but in some cases it does require a member of the team to review the application. This is of course happens at the point where an application was submitted and is not in progress or draft (the preliminary checks). Hope your application has been processed by now 👍 If not let us know.
